pkg/enums/controlstatus.go

```diff
@@ -0,0 +1,74 @@
+package enums
+
+import (
+	"fmt"
+	"io"
+	"strings"
+)
+
+type ControlStatus string
+
+var (
+	// ControlStatusPreparing indicates the control is being prepared
+	ControlStatusPreparing ControlStatus = "PREPARING"
+	// ControlStatusNeedsApproval indicates the control needs to be approved before it is available
+	ControlStatusNeedsApproval ControlStatus = "NEEDS APPROVAL"
+	// ControlStatusChangesRequested indicates the control was rejected and needs some changes to be approved
+	ControlStatusChangesRequested ControlStatus = "CHANGES REQUESTED"
```

Comment:
```suggestion
	ControlStatusNeedsApproval ControlStatus = "NEEDS_APPROVAL"
	// ControlStatusChangesRequested indicates the control was rejected and needs some changes to be approved
	ControlStatusChangesRequested ControlStatus = "CHANGES_REQUESTED"
```

Can you allow `null` as a valid value here as well - that should be the default on creation - only when controls are within an org would we have a status